examples/web_ws.py

```diff
@@ -18,8 +20,8 @@ async def wshandler(request: web.Request) -> Union[web.WebSocketResponse, web.Re
     resp = web.WebSocketResponse()
     available = resp.can_prepare(request)
     if not available:
-        with open(WS_FILE, "rb") as fp:
-            return web.Response(body=fp.read(), content_type="text/html")
+        async with aiofiles.open(WS_FILE, "rb") as fp:
+            return web.Response(body=await fp.read(), content_type="text/html")
```

Comment:
I think this is reasonable (although I'd note that we seem to use `to_thread()` or similar in our internal code).